dev/com.ibm.ws.jpa_spec10_injection_fat/fat/src/com/ibm/ws/jpa/RepeatWithJPA20.java

```diff
@@ -26,19 +23,10 @@
  *
  */
 public class RepeatWithJPA20 extends FeatureReplacementAction {
```

Comment:
this class should just implement `RepeatTestAction` like the other ones, instead of extending `FeatureReplacementAction`

Reply:
Ah, good catch there.